### PR TITLE
Fix GH action workflow for notifying incoming webhooks on community/hub servers

### PR DESCRIPTION
**Pull request by mickmister**
#### Summary

This PR uses newly set environment variables with incoming webhooks to send release updates to the community/hub servers.

#### Ticket Link

Fixes https://mattermost.atlassian.net/browse/CLD-7673

https://hub.mattermost.com/private-core/pl/cdque38knirqpet1soc8x1398e

### CODE REVIEW

**Review by hanzei — APPROVED** [review]
Thanks for the fix :+1:

File: .github/workflows/notify-mm-blog.yml [hunk 1]
```diff
@@ -17,12 +17,12 @@ jobs:
         shell: bash
         run: |
           .github/workflows/notify-mm-blog.sh
-      - name: Notify channel 1
+      - name: Notify "Community Configuration" channel on community
         uses: mattermost/action-mattermost-notify@master
```
Comment (hanzei) [inline .github/workflows/notify-mm-blog.yml:21]: Should we pin the version?

**Review by hanzei — CHANGES_REQUESTED** [review]
Let's merge https://github.com/mattermost/mattermost-marketplace/pull/441 first

**Comment by mickmister** [comment]
Closing @hanzei

**Pull request closed without merging**